docs/rest-apis/collections/methods/get-collections.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+parent: Collections
+title: Get Collections
+permalink: /collections/get-collections
+nav_order: 2
+---
+
+
+## HTTP Request
+
+```
+GET https://accounts.workfloplus.com/collections/v1/collection
```

Comment:
should be gateway not accounts in the URL, think this is across most of the collections files

Reply:
Added {{ site.main_api_base_url }} instead of accounts.workfloplus.com